runtime/src/bank.rs

```diff
@@ -5713,6 +5713,25 @@ impl Bank {
             .map_or(Ok(()), |sig| self.get_signature_status(sig).unwrap())
     }
 
+    /// Process a Transaction. This is used for unit tests that exercise program log
+    /// subscriptions. It simply calls the vector Bank::process_transactions method.
```

Comment:
```suggestion
    /// Process a Transaction and store program log data. This is used for unit tests, and simply
    /// replicates the vector Bank::process_transactions method with `enable_cpi_recording: true`
```
There might be another use-case for this function than subscriptions. In fact, it would kind of rad to extend our coverage of the log recorder (note to self)